src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react'
+import BootstrapAlert from 'react-bootstrap/Alert'
+import Button from 'react-bootstrap/Button'
+
+interface Props {
+  /** Defines the color of the alert. Defaults to primary.
+   * @default "primary"
+   **/
+  color?: 'primary' | 'secondary' | 'success' | 'warning' | 'danger' | 'info' | 'light' | 'dark'
```

Comment:
The most nit pick comment ever: 

What do you think about having the docs read like: 

```js
/**
* @default "primary"
* Defines the color of the alert. Defaults to primary.
*/
```

Or is this how prettier formatted it?

Reply:
No problem, we just have to select one implementation. Prettier doesn't care.

Reply:
Well, actually I just found out that the `@default` value must be set _after_ the description.
Let's make this comments four lines long but with description first, ok?